cmd/livenessprobe/main.go

```diff
@@ -119,16 +124,23 @@ func acquireConnection(ctx context.Context, metricsManager metrics.CSIMetricsMan
 }
 
 func main() {
-	klog.InitFlags(nil)
-	flag.Set("logtostderr", "true")
+	fg := featuregate.NewFeatureGate()
```

Comment:
considering the amount of code that needs to be shared maybe it's ok to have it in this repo and not in csi-lib-utils